stores/mongodb/src/vector/index.test.ts

```
@@ -1,5 +1,6 @@
 import { createVectorTestSuite } from '@internal/storage-test-utils';
 import { vi, describe, it, expect, beforeAll, afterAll, test } from 'vitest';
+import { MongoClient } from 'mongodb';
```

ESLint

[error] `mongodb` import should occur before import of `vitest`

(import/order)